packages/block-library/src/gallery/index.js

```diff
@@ -95,47 +142,39 @@ export const settings = {
 				);
 			},
 		},
-		{
```

Comment:
The previous definition is nearly the same (it only adds one optional attribute in HTML output) making this entry redundant. It can be safely removed.